### PR TITLE
8323221: Create release notes for JavaFX 21.0.2

### PR DESCRIPTION
**Pull request by kevinrushforth**
Release notes for JavaFX 21.0.2.

Notes to reviewers:

I used the following filter to pick the issues:

https://bugs.openjdk.org/issues/?filter=45268

The original filter, with the backport IDs, is here:

https://bugs.openjdk.org/issues/?filter=45267

As usual, I excluded test bugs, cleanup bugs, etc.

NOTE: Once the CPU release ships on 2023-01-16 (tomorrow), I will add any security bugs and make this PR `rfr`.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] [JDK-8323221](https://bugs.openjdk.org/browse/JDK-8323221) needs maintainer approval
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8323221](https://bugs.openjdk.org/browse/JDK-8323221): Create release notes for JavaFX 21.0.2 (**Task** - P2 - Approved)

### Reviewers
 * @abhinayagarwal (no known openjdk.org user name / role)
 * [Johan Vos](https://openjdk.org/census#jvos) (@johanvos - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx21u.git pull/39/head:pull/39` \
`$ git checkout pull/39`

Update a local copy of the PR: \
`$ git checkout pull/39` \
`$ git pull https://git.openjdk.org/jfx21u.git pull/39/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 39`

View PR using the GUI difftool: \
`$ git pr show -t 39`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx21u/pull/39.diff">https://git.openjdk.org/jfx21u/pull/39.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx21u/pull/39#issuecomment-1894194404)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
Reviewers: @johanvos @abhinayagarwal

**Comment by kevinrushforth** [comment]
/approval request These are the release notes for a JavaFX 21.0.x release, so are needed in jfx21u.

**Comment by abhinayagarwal** [comment]
Looks good. I will review this again once the PR has been labelled as `rfr`.

**Comment by kevinrushforth** [comment]
This is now ready for review.

@johanvos in addition to review I need maintainer approval.

**Review by abhinayagarwal — APPROVED** [review]

**Review by johanvos — APPROVED** [review]

**Pull request closed without merging**
